src/tech/how-we-do-development.md

```diff
@@ -411,4 +411,24 @@ maintainers and contributors understand the reasoning for not doing so.
 
 See
 [Tech Team RFC-019](https://github.com/dxw/tech-team-rfcs/blob/main/ways-of-working/rfc-019-use-changelogs-to-track-changes.md)
-for more context around this decision.
\ No newline at end of file
+for more context around this decision.
+
+## Developing alone
+
+Developers at senior level and above are expected to be able to be the sole technologist on a project. This can uncover issues with the commit/review/merge cycle, especially if the project is beyond the discovery or alpha phase, as there is no developer to collaborate with and review pull requests.
```

Comment:
"commit/review/merge cycle" -> this seems more focused on reviewing specifically. Are there impacts on commits and merges as well?

Reply:
There are impacts on merges when a client's security policy disallows developers merging their own commits (DfE does this for example). I'll take out "commit" though, you're right there.

Reply:
You're right I'm getting confused with approvals with write access vs approvals with no write access 🤦